src/rng.cpp

```diff
@@ -1,19 +1,30 @@
-#include "output.h"
 #include "rng.h"
-#include <stdlib.h>
+#include <random>
 
-long rng(long val1, long val2)
+namespace {
+std::mt19937 random_generator {std::random_device{}()};
```

Comment:
Sure, if consistency across standard libraries isn't a big concern, that change is any easy one.
AFAIK the implementations used are:

Platform | Implementation
--- | --- 
Visual C++ | mt19937
GCC 4.8(libstdc++) | minstd_rand0
Clang 3.4(libc++) | minstd_rand